stl/inc/tempCodeRunnerFile.cpp

```diff
@@ -0,0 +1 @@
+_Iterator_base12* _Mynextiter{};

```

Comment:
I think you didn't mean to add this file?

Reply:
No ,  Will remove this.